File: integration/feature/test-quick/resources/app/test/src/MyStringCombinatorTests.scala
package app

import utest.*
import app.MyString

object MyStringCombinatorTests extends TestSuite {
  def tests = Tests {
    test("simple") {
      val a = MyString("a")
      val b = MyString("b")
      val c = MyString("c")
      val result = MyString.combine(a, b, c)
      assert(result == MyString("abc"))
    }
  }
}

File: integration/feature/test-quick-java/resources/app/test/src/MyStringCombinatorTests.java
package app;

import org.junit.Test;
import static org.junit.Assert.*;

public class MyStringCombinatorTests {
    @Test
    public void simple() {
        MyString a = new MyString("a");
        MyString b = new MyString("b");
        MyString c = new MyString("c");
        MyString result = MyString.combine(a, b, c);
        assertEquals(new MyString("abc"), result);
    }
}
